docs/data/date-pickers/date-calendar/date-calendar.md

```diff
@@ -69,12 +69,11 @@ You can also customize what's rendered as a calendar week number, using a callba
 
 {{"demo": "AddWeekNumber.js"}}
 
-### Custom day rendering
+### Week picker
```

Comment:
Would it make sense to add a redirect rule for this URL change?

Reply:
For me we can't redirect the hash part of an URL